src/concept-coach/error-notification.cjsx

```diff
@@ -43,20 +43,25 @@ ErrorNotification = React.createClass
         {for error, i in @state.errors
           <li key={i}>{error}</li>}
       </ul>
+      <p>
+        {window.navigator.userAgent}
+      </p>
     </BS.Panel>
 
   render: ->
     return null unless @state.errors
-    <BS.Modal className='errors' onRequestHide={@onHide} title="Server Error">
+    <BS.Modal className='errors' onRequestHide={@onHide} title="Error encountered">
       <div className='modal-body'>
-        <h3>Server error encountered</h3>
         <p>
           An unexpected error has occured.  Please
           visit <a target="_blank"
-            href="https://openstaxtutor.zendesk.com/hc/en-us/requests/new"
+            href="https://openstaxcc.zendesk.com/hc/en-us"
           > our support site </a> so we can help to diagnose and correct the issue.
         </p>
-        <BS.Button onClick={@toggleDetails}>
+        <p>
+          When reporting the issue, it would be helpful if you could include the error details.
+        </p>
+        <BS.Button className='-display-errors' style={marginBottom: '1rem'} onClick={@toggleDetails}>
           {if @state.isShowingDetails then "Hide" else "Show"} Details
```

Comment:
Yeah, we could do that but then it's a stray style hanging out there and will be forgotten.  Then when/if we ever remove the button the style won't be remembered and will remain in the main stylesheet forever.  

There's tons of opinions on the validity of using inlines styles with react.  I'm slowly coming to the opinion that inline styles are a good for a single style like the above when the component doesn't have an associated stylesheet.  When the component gets more than one or two inline styles then they should be copied to a stylesheet.

But like I said, I'm still in flux on the subject, and I pointed it out was to gather other viewpoints.